src/status_im/ui/screens/chat/message/message_old.cljs

```diff
@@ -86,28 +86,51 @@
          :accessibility-label :message-timestamp}
         timestamp-str]])))
 
+
 (defview quoted-message
-  [_ {:keys [from parsed-text image]} outgoing current-public-key public? pinned]
+  [_ {:keys [from parsed-text image audio sticker id] :as message} outgoing current-public-key public? pinned chat-id]
   (letsubs [contact-name [:contacts/contact-name-by-identity from]]
-    [react/view {:style (style/quoted-message-container (and outgoing (not pinned)))}
-     [react/view {:style style/quoted-message-author-container}
-      [chat.utils/format-reply-author
-       from
-       contact-name
-       current-public-key
-       (partial style/quoted-message-author (and outgoing (not pinned)))
-       (and outgoing (not pinned))]]
-     (if (and image
-              ;; Disabling images for public-chats
-              (not public?))
-       [fast-image/fast-image {:style  {:width            56
-                                        :height           56
-                                        :background-color :black
-                                        :border-radius    4}
-                               :source {:uri image}}]
-       [react/text {:style           (style/quoted-message-text (and outgoing (not pinned)))
+           [react/view {:style (style/quoted-message-container (and outgoing (not pinned)))}
+            [react/view {:style style/quoted-message-author-container}
+             [chat.utils/format-reply-author
+              from
+              contact-name
+              current-public-key
+              (partial style/quoted-message-author (and outgoing (not pinned)))
+              (and outgoing (not pinned))]]
+            (if (and image
+                     ;; Disabling images for public-chats
+                     (not public?))
+              [fast-image/fast-image
+               {:style  {:width            56
+                         :height           56
+                         :background-color :black
+                         :border-radius    4}
+                :source {:uri image}}]
+
+              (if (and audio
+                       ;; Disabling audio for public-chats
+                       (not public?))
+                [react/view {:style (style/message-view message) :accessibility-label :audio-message}
+                 [react/view {:style (style/message-view-content)}
+                  [react/view {:style (style/style-message-text outgoing)}
+                   [message.audio/message-content message]
+                   [message-status message]]]]
+
+                (if sticker
+                  (if (def repliedMsg (@(re-frame/subscribe [:chats/chat-messages chat-id]) id))
```

Comment:
- Please move subscription to letsubs
- Also `(def ...)` returns nil, so `(if (def ...))` will be always false
- @(re-frame/subscribe ...) will return message object and by surrounding this with (), you are using this as a function, wouldn't that will throw error?

Reply:
- The line `(if (def repliedMsg (@(re-frame/subscribe [:chats/chat-messages chat-id]) id))` will be true if the repliedMsg exists (has not been deleted), otherwise will be false. I have refactored it to `letsubs` anyway.
- `(@(re-frame/subscribe [:chats/chat-messages chat-id]) id)` is used to get a particular message (the replied-to message), hence the parentheses are needed. Actually removing the parentheses would throw an error.